benchmarks/distributed/lambda_runtimes/cpp_runtime/lambda_function.py

```diff
@@ -0,0 +1,13 @@
+import json
+import subprocess
+import os
+import time
+import cloudpickle
+import base64
+
+
```

Comment:
What does this file do? Please add some comments. Where is it called/used in the overall setup? I.e., a python lambda function.py part of cpp_runtime? Is that correct?